docs/driver-only-builds.md

```diff
@@ -145,9 +145,9 @@ timeline, please let us know if you're interested.
 
 ### Limitations regarding the selection of curves
 
-There is an ongoing work which tries to establish a link/constrain between
-the list of supported curves and supported algorithms both in the builtin and
-PSA sides. In particular:
+There is ongoing work which is trying to establish the links and constraints
```

Comment:
You could just say "There is ongoing work to establish the links and constraints..."

Reply:
Agree, prefer this wording

Reply:
I'll point out this sentence is going to be changed again in #7757 once this PR and #8075 are in, so I don't think it's essential to get the wording perfect right now (we can be much more picky in the PR the implements #7757).